src/main/java/Duke.java

```diff
@@ -1,10 +1,124 @@
+                try {
+                    String[] arrOfStr = input.split(" ", 2);
+                    tasks.add(new Todo(arrOfStr[1]));
+                    counter = counter + 1;
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(tasks.get(counter - 1));
+                    if (counter == 1) {
+                        System.out.println("Now you have " + String.valueOf(counter) + " task in the list.");
+                    } else {
+                        System.out.println("Now you have " + String.valueOf(counter) + " tasks in the list.");
+                    }
+
+                } catch(ArrayIndexOutOfBoundsException e) {
+                    throw new DukeException("☹ OOPS!!! The description of a todo cannot be empty.");
+                }
+
+            } else if(input.length() > 7  && (input.substring(0, 8).equals("deadline"))) {
+                try {
+                    String[] arrOfStr = input.split(" ", 2);
+                    String[] arrOfStr2 = arrOfStr[1].split("/", 2);
+                    String[] arrOfStr3 = arrOfStr2[1].split(" ", 2);
+
+                    tasks.add(new Deadline(arrOfStr2[0], arrOfStr3[1]));
+                    counter = counter + 1;
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(tasks.get(counter - 1));
+                    if (counter == 1) {
+                        System.out.println("Now you have " + String.valueOf(counter) + " task in the list.");
+                    } else {
+                        System.out.println("Now you have " + String.valueOf(counter) + " tasks in the list.");
+                    }
+
+                } catch(ArrayIndexOutOfBoundsException e) {
+                    throw new DukeException("☹ OOPS!!! Please provide a deadline for your task.");
+
+                }
+            } else if(input.length() > 4 && input.substring(0, 5).equals("event")) {
+                try {
+                    String[] arrOfStr = input.split(" ", 2);
+                    String[] arrOfStr2 = arrOfStr[1].split("/", 2);
+                    String[] arrOfStr3 = arrOfStr2[1].split(" ", 2);
+
+                    tasks.add(new Event(arrOfStr2[0], arrOfStr3[1]));
+                    counter = counter + 1;
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(tasks.get(counter - 1));
+                    if (counter == 1) {
+                        System.out.println("Now you have " + String.valueOf(counter) + " task in the list.");
+                    } else {
+                        System.out.println("Now you have " + String.valueOf(counter) + " tasks in the list.");
+                    }
+                } catch(ArrayIndexOutOfBoundsException e) {
+                    throw new DukeException("☹ OOPS!!! Please provide a timing for your event.");
+                }
```

Comment:
Perhaps this might have been missed during checks, did you made a mistake and left out a empty white line to separate the if-else logic blocks?